pages/events/summer-school-2023.vue

```diff
@@ -26,13 +26,7 @@
           for more details. For any questions, please check out our FAQ below!
         </p>
         <p>See you soon!</p>
-        <UiCta
-          class="summer-school-page__cta summer-school-page__cta_disabled"
-          :url="header.cta.url"
-          :label="header.cta.label"
-          :segment="header.cta.segment"
-          kind="secondary"
-        />
+        <UiCta class="summer-school-page__cta" v-bind="header.cta" />
```

Comment:
With the pull request https://github.com/Qiskit/qiskit.org/pull/3327, we replaced the uses of `v-bind`.

Here's a example of the change required for a **UiCta** component: https://github.com/Qiskit/qiskit.org/pull/3327/files#diff-67f6e6ba781d83d1e242ee68b12b45ac6c5c7aecf7d0ffd115ecf2548aa40bbbR28-R33

Reply:
Thanks for the review! I addressed your comment in the latest commit